homeassistant/components/squeezebox/sensor.py

```diff
@@ -0,0 +1,110 @@
+"""Platform for sensor integration for squeezebox."""
+
+from __future__ import annotations
+
+import logging
+from typing import cast
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import EntityCategory, UnitOfTime
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from . import SqueezeboxConfigEntry
+from .const import (
+    STATUS_SENSOR_INFO_TOTAL_ALBUMS,
+    STATUS_SENSOR_INFO_TOTAL_ARTISTS,
+    STATUS_SENSOR_INFO_TOTAL_DURATION,
+    STATUS_SENSOR_INFO_TOTAL_GENRES,
+    STATUS_SENSOR_INFO_TOTAL_SONGS,
+    STATUS_SENSOR_LASTSCAN,
+    STATUS_SENSOR_NEWPLUGINS,
+    STATUS_SENSOR_NEWVERSION,
+    STATUS_SENSOR_OTHER_PLAYER_COUNT,
+    STATUS_SENSOR_PLAYER_COUNT,
+)
+from .entity import LMSStatusEntity
+
+SENSORS: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_ALBUMS,
+        state_class=SensorStateClass.TOTAL,
+        icon="mdi:album",
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_ARTISTS,
+        state_class=SensorStateClass.TOTAL,
+        icon="mdi:account-music",
+    ),
+    SensorEntityDescription(
+        key=STATUS_SENSOR_INFO_TOTAL_DURATION,
+        state_class=SensorStateClass.TOTAL,
+        device_class=SensorDeviceClass.DURATION,
+        native_unit_of_measurement=UnitOfTime.SECONDS,
+    ),
```

Comment:
Should we add a suggested unit of measurement

Reply:
It seems to be scaling nicely Test Server has a smallish collections and Prod one is large.

Reply:
I get that, but knowing the duration in seconds is a bit, strange, hours seems a more logical unit

Reply:
Well you get S:M:H's displayed on the sensor and seconds in the graph so I think in reasonable and allows your to automate easily on small changes

Reply:
Its worth noting we get length down to the sub-second from LMS this is the sum of all tracks

Reply:
Oh I would expect it would just display 3254352345 seconds or something